packages/@glimmer/core/test/non-interactive/render-test.ts

```diff
@@ -165,28 +166,26 @@ module(`[@glimmer/core] non-interactive rendering tests`, () => {
   //   assert.equal(await render(MainComponent), 'Hello Glimmer!');
   // });
 
-  // test('a component can inject services', async assert => {
-  //   class LocaleService {
-  //     get currentLocale(): string {
-  //       return 'en_US';
-  //     }
-  //   }
+  test('components recieve owner', async assert => {
+    class LocaleService {
+      get currentLocale(): string {
+        return 'en_US';
+      }
+    }
 
-  //   class MyComponent extends Component {
-  //     get myLocale(): string {
-  //       return (getHostMeta(this) as { locale: LocaleService })!.locale.currentLocale;
-  //     }
-  //   }
+    class MyComponent extends Component {
+      get myLocale(): string {
+        return getOwner<LocaleService>(this).currentLocale;
+      }
+    }
 
-  //   setComponentTemplate(MyComponent, createTemplate('<h1>{{this.myLocale}}</h1>'));
+    setComponentTemplate(MyComponent, createTemplate('<h1>{{this.myLocale}}</h1>'));
 
-  //   const html = await render(MyComponent, {
-  //     meta: {
-  //       locale: new LocaleService(),
-  //     },
-  //   });
-  //   assert.strictEqual(html, '<h1>en_US</h1>');
-  // });
+    const html = await render(MyComponent, {
+      owner: new LocaleService(),
```

Comment:
I think it would be generally better to keep this test somewhat representative of what we expect folks to do.

I'm thinking:

```suggestion
      owner: { 
        services: {
         locale: new LocaleService(),
        },
      },
```